src/vs/base/test/common/event.test.ts

```diff
@@ -177,6 +177,33 @@ suite('Event',function(){
 			Errors.setUnexpectedErrorHandler(origErrorHandler);
 		}
 	});
+
+	test('Debounce Event', function () {
```

Comment:
since this test is async (setTimeout) you must add the `done` parameter and call it. Alternatively make this test return a promise.